README.md

```diff
@@ -87,7 +88,8 @@ For more maps created with QGIS, visit the [QGIS Map Showcase Flickr Group](http
 
 - Fully customizable user experience, including user interface and application settings that cater to power-users and beginners alike
 - Rich [expression engine](https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/expression.html) for maximum flexibility in visualization and processing
-- Broad and varied [plugin ecosystem](https://plugins.qgis.org/) that includes data connectors, digitizing aids, advanced analysis and charting tools, in-the-field data capture, etc.
+- Broad and varied [plugin ecosystem](https://plugins.qgis.org/) that includes data connectors, digitizing aids, advanced analysis and charting tools, 
+in-the-field data capture, conversion of ESRI style files, etc.
```

Comment:
Not sure how good that would be but SLYR supports importing projects made in argis (mxd, mxt & pmf). Now this require a subscription, but it might be nice to mention?

Reply:
Think best for now to limit explicit mention of what is available in free plugins (including community edition of SLYR) and go back and add project import when that moves to community-SLYR. Also, though it would be quite the stretch in logic, I also think it's better to not even give the possible impression of preferentially "marketing" subscription tools by core QGIS developers in the readme file.

That all said, this is not functionality I use, and I welcome better wording suggestions!

Reply:
I agreee that it's a fine line. Might be good to simply my mention support of esri files/formats ( not mentionning project or styles) but leaving it open?

Reply:
Pinging @nyalldawson who might have an opinion here. (He's a busy guy, so if he doesn't respond to this in a few days we'll just do something sensible; it's not critical path.)

Reply:
I'm fine either way. Fwiw vector lyr support will be added to the community version when I'm back at work later this month!

Reply:
OK thanks Nyall (deliberately not pinging if you're on vacation(?)). I think best to leave for now as I have it, "ESRI style files", since projects not yet free, and just saying "esri files/formats" can generate confusion, e.g. "oh, do I need a plug in to read esri shapefiles?"  I do promise to come back and add "and project files" once that is part of community version too...

Reply:
Sounds good to me!